.github/fabricbot.json

```diff
@@ -3006,6 +3006,130 @@
       ]
     }
   },
+  {
+    "taskSource": "fabricbot-config",
+    "taskType": "trigger",
+    "capabilityId": "IssueResponder",
+    "subCapability": "IssuesOnlyResponder",
+    "version": "1.0",
+    "config": {
+      "taskName": "Add to AppModel Project",
+      "actions": [
+        {
+          "name": "addToProject",
+          "parameters": {
+            "projectName": "AppModel",
```

Comment:
This needs to be an exact string match for the project name. Should this be "AppModel .NET 7"?

Reply:
I actually renamed the project to "AppModel" so hopefully that worked.